app.config.ts

```diff
@@ -11,7 +11,9 @@ const buildNumber: string = getValue(
   process.env.APP_BUILD_VERSION,
   "Please provide a Build Version with APP_BUILD_VERSION env variable"
 );
-const androidVersionCode: number = parseInt(buildNumber);
+// The APP_BUILD_VERSION number is 38 behind the actual deploy, adding 38 synchronises the build number
+const androidVersionCode = parseInt(buildNumber) + 38;
+const appBuildVersion = androidVersionCode.toString();
```

Comment:
This part's a bit clunky, I suggest to name it as the following, then only in the `ios.buildNumber` you directly call `.toString`
```suggestion
const appBuildVersion = parseInt(buildNumber) + 38;
```